OpenUpSA/wazimap-ng-ui
# translations for the point data related texts

## Description
adding `i18n` class to necessary divs to be able to update point data related texts using the profile configuration

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-74


## How is it tested automatically?
created translation.feature

## How should a reviewer test it locally
* pick a translatable key from https://app.gitbook.com/o/-Ly4_0FbvQFY-m3TOzuc/s/-M3zapxOsLjq_qKRcmH8/profile-configuation#translatable-keys
* set it in the profile config `translations` property
* on the website confirm that the text is changed

## Screenshots
before : 
![image](https://user-images.githubusercontent.com/53019884/192159932-c2388b85-10bc-4b7f-a454-b6215e2412eb.png)

after :
![image](https://user-images.githubusercontent.com/53019884/192159936-d4e6b732-b807-45e4-bd60-b096595a269e.png)



## Changelog

### Added
* `__tests__/gui/translation.feature` to test the feature

### Updated
* `src/js/webflow/import.js` to add `i18n` class to the necessary elements

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
